graphdatascience/query_runner/progress/query_progress_logger.py

```diff
@@ -65,14 +65,27 @@ def _log(
                 root_task_name = task_with_progress.task_name
                 progress_percent = task_with_progress.progress_percent
 
-                if progress_percent == "n/a":
-                    return
-
-                if not pbar:
-                    pbar = tqdm(total=100, unit="%", desc=root_task_name, maxinterval=self._LOG_POLLING_INTERVAL)
-
-                parsed_progress = float(progress_percent[:-1])
-                pbar.update(parsed_progress - pbar.n)
+                has_relative_progress = progress_percent != "n/a"
```

Comment:
nit: Its generally easier to understand boolean logic if negations can be avoided: `has_unknown_progress = progress_percent == "n/a"` and then flip the logic below